process/txsimulator/simulationAccountsDB_test.go

```diff
@@ -17,15 +17,15 @@ import (
 func TestNewReadOnlyAccountsDB_NilOriginalAccountsDBShouldErr(t *testing.T) {
 	t.Parallel()
 
-	roAccDb, err := NewReadOnlyAccountsDB(nil)
+	roAccDb, err := NewSimulationAccountsDB(nil)
```

Comment:
rename the variable to accountsDB ? (ro came from read-only)
Valid for all tests

Reply:
done